test/src/routers/controllers/addPresenterController.test.ts

```diff
@@ -15,7 +15,7 @@ const router = supertest(app);
 const en = require("../../../../src/locales/en/translation/add-presenter.json");
 const cy = require("../../../../src/locales/cy/translation/add-presenter.json");
 const companyNumber = "12345678";
-const urlwithCompNum = "/your-companies/company/:companyNumber/add-presenter";
+const urlwithCompNum = "/your-companies/add-presenter/:companyNumber";
```

Comment:
what url is add-presenter?

Reply:
is it our page?

Reply:
We usually follow the URLs in the prototype so Brendan grabbed it from there. It's the add new authorised person URL. I have no idea why it's named this way. Usually the URLs are fairly sensible. Here is a link to the prototype https://find-update-company-info.herokuapp.com/v6/add-presenter